### PR TITLE
Fixes overwatch consoles on WO 

### PR DESCRIPTION
**Pull request by TheGamerdk**
# About the pull request

There is no OB cannon on WO

# Explain why it's good for the game
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
fix: Overwatch now works on Whiskey Outpost again, not that you will use it.
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by mullenpaul — APPROVED** [review]

File: tgui/packages/tgui/interfaces/OverwatchConsole.jsx [hunk 1]
```diff
@@ -87,12 +87,14 @@ const SquadPanel = (props, context) => {
...
-          Orbital Bombardment
-        </Tabs.Tab>
+        {!!data.can_launch_obs && (
+          <Tabs.Tab
+            selected={category === 'ob'}
+            icon="bomb"
+            onClick={() => setCategory('ob')}>
+            Orbital Bombardment
+          </Tabs.Tab>
+        }}
```
Comment (mullenpaul) [inline tgui/packages/tgui/interfaces/OverwatchConsole.jsx:97]: Not closing this proper - you need to close like )} not }}

Comment (TheGamerdk) [reply tgui/packages/tgui/interfaces/OverwatchConsole.jsx:97]: copy paste gone wrong (i made this UI originally should've caught that, oops)

**Review by harryob — APPROVED** [review]

**Pull request merged**